### DISCUSSION TITLE
外网浏览器可以访问ddns后的alist，但是nplayer中无法访问

### DISCUSSION THREAD

**Discussion by dage1997** [AlistGo/alist#4940] [Q&A]
在外网用ddns后的域名或者公网ip直接在浏览器访问alist都没有问题，用公网ip在nplayer访问也没有问题。但是但是用ddns的域名通过webdav挂载到nplayer就显示Sever returned nothing。![image](https://github.com/alist-org/alist/assets/63339214/3777be96-a5a7-4df5-9e83-8cabc231128c)

**Comment by SeanHeuc** [comment]
url最后要加上/dav 才能访问webdav
